gemrb/core/GUI/GameControl.cpp

```diff
@@ -2163,6 +2188,7 @@ bool GameControl::OnMouseUp(const MouseEvent& me, unsigned short Mod)
 			ClearMouseState();
 			return true;
 		}
+		InitFormation(p);
```

Comment:
is this not redundant with `ClearMouseState`?

Reply:
No, this ensures that we get a proper rotation even for left-click moves. I'm adding a comment.